boards/arduino-mkrfox1200/doc.txt

```diff
@@ -16,19 +16,14 @@ powered by an Atmel SAMD21 microcontroller.
 
 ### Flash the board
 
-1. Put the board in bootloader mode by double tapping the reset button.<br/>
-   When the board is in bootloader mode, the user led (green) oscillates
-   smoothly.
```

Comment:
How about keeping this info around for when the firmware crashed and the regular option doesn't work anymore.

(Unfortunately this is relevant for all the board documentation here)

Reply:
added a note about that in the documentation of each affected boards.